package.json

```diff
@@ -53,7 +53,38 @@
           "description": "Run the linter on save (onSave) or on type (onType)."
         }
       }
-    }
+    },
+    "languages": [
+      {
+        "id": "json",
+        "filenames": [
+          ".pug-lintrc",
+          ".jade-lintrc"
+        ]
+      }
+    ],
+    "jsonValidation": [
+      {
```

Comment:
Unfortunately I had to write the `"jsonValidation"` array this way because `fileMatch` can't be a glob.